### PR TITLE
feat(cli): KICS does not execute scan as default anymore

### LINKED ISSUE #3029: KICS does not execute scan as default
### Description
KICS should not execute scan as default, so it will be updated to respect this behavior

### PR DESCRIPTION
**Pull request by felipe-avelar**
Signed-off-by: Felipe Avelar <felipe.avelar@checkmarx.com>

Closes #3029 

**Proposed Changes**
- Kics does not execute scan as default anymore
- Removing deprecation warning

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: internal/console/kics_test.go [hunk 1]
```diff
@@ -90,6 +90,13 @@ func TestConsole_Execute(t *testing.T) { //nolint
 			wantErr: false,
 			remove:  "",
 		},
+		{
+			name: "test_kics_fail_without_scan",
+			args: []string{"kics", "--path", filepath.FromSlash("../../test/fixtures/tc-sim01/positive1.tf"),
```
Comment (rogeriopeixotocx) [inline internal/console/kics_test.go:95]: nice

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Comment by milesflo** [comment]
Please accept this PR to get this change pushed into the runner as **soon as possible.** https://github.com/Checkmarx/kics-github-action/pulls

**Pull request merged**